bindings/matrix-sdk-ffi/src/room_info.rs

```diff
@@ -63,7 +63,7 @@ impl RoomInfo {
             joined_members_count: room.joined_members_count(),
             highlight_count: unread_notification_counts.highlight_count,
             notification_count: unread_notification_counts.notification_count,
-            notification_mode: room.notification_mode().await.map(Into::into),
+            notification_mode: room.notification_mode(true).await.map(Into::into),
```

Comment:
It's not really clear what `(true)` means as an argument list for `.notification_mode()`. Maybe we should have separate `notification_mode` and `user_defined_notification_mode` or `notification_mode` and `effective_notification_mode`?

Reply:
Ok, I've added a separate `user_defined_notification_mode()` function.